### DISCUSSION TITLE
150

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#89] [General]
# 潮流周刊第150期 - 重庆雄起

封面图来源于周末去重庆下机后机场外拍的，玩了 2 天挺不错的，感谢招待的小伙伴，我这次尽量不去网红打卡点，到处随便逛逛吃吃，挺舒服的。

https://weekly.tw93.fun/posts/150-%E9%87%8D%E5%BA%86%E9%9B%84%E8%B5%B7

**Comment by cqmzgg** [comment]
# 重庆欢迎你！！
哈哈，这是机场的接车队伍。。

**Reply by tw93** [reply]
哈哈哈哈哈哈 牛牛牛

**Comment by zhuhaoxlj** [comment]
白板 https://revezone.com/ 也很好用🎉
